primitives/rialto/src/lib.rs

```diff
@@ -34,6 +34,14 @@ use sp_runtime::{
 };
 use sp_std::prelude::*;
 
+/// Number of extra bytes (excluding size of storage value itself) of storage proof, built at
+/// Rialto chain. This mostly depends on number of entries (and their density) in the storage trie.
+/// Some reserve is reserved to account future chain growth.
+pub const EXTRA_STORAGE_PROOF_SIZE: u32 = 1024;
```

Comment:
Actually this is ~800 bytes for our almost empty chains. I was pretty sure that it is will be much lesser.